tests/active_learning/test_ScorerClassification.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import numpy as np
+
+from lightly.active_learning.scorers.classification import ScorerClassification
+
+
+class TestScorerClassification(unittest.TestCase):
+
+    def test_score_calculation(self):
+        n_samples = 100
+        n_classes = 10
+        predictions = np.random.rand(n_samples, n_classes)
+        predictions_normalized = predictions / np.sum(predictions, axis=1)[:, np.newaxis]
+        model_output = predictions_normalized
+        scorer = ScorerClassification(model_output)
+        scores = scorer._calculate_scores()
+        scores_prediction_entropy = scores["prediction-entropy"]
+        scores_prediction_margin = scores["prediction-margin"]
+
+        assert scores_prediction_entropy.shape == (n_samples, )
+        assert scores_prediction_margin.shape == (n_samples, )
+        assert all(scores_prediction_entropy > 0)
```

Comment:
This assertion will fail for the following example input:
```
[
   [1.0, 0.0, 0.0, 0.0, 0.0]
]
```
because the logarithm retains the original value for all values <= 0.